pkg/nvml/nvml.go

```diff
@@ -275,6 +275,15 @@ func nvmlDeviceGetSerial(nvmlDevice nvmlDevice, Serial *byte, Length uint32) Ret
 	return __v
 }
 
+// nvmlDeviceGetModuleId function as declared in nvml/nvml.h
+func nvmlDeviceGetModuleId(nvmlDevice nvmlDevice, ModuleId *uint32) Return {
```

Comment:
Note to self: Can we update `c-for-go` to return `moduleId` here instead of `ModuleId`?

Reply:
I'm not sure. We could / should look into it.